### PR TITLE
Rating and grid components

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Added rating and grid components.

Rating link: https://ym-cl-rating-and-grid.netlify.app/components/rating/rating
Grid link: https://ym-cl-rating-and-grid.netlify.app/components/grid/grid

### CODE REVIEW

**Review by Sahil25061999 — APPROVED** [review]
just a suggestion you can just create a separate class for the grid property itself rather than giving it in grid-2

File: components/rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        sandbox="allow-scripts allow-same-origin">
+        </iframe>
+    </div>
+
+    <script>
+        const starOne = document.getElementById('star-one');
```
Comment (pradipgharge) [inline components/rating/rating.html:36]: It would be good to have a separate js file for this !

**Review by pradipgharge — COMMENTED** [review]
Code looks good, just add the javascript in separate file.

File: components/rating/rating.html (hunk 1 above)
Comment (yaswanthmyneni) [reply components/rating/rating.html:36]: did it in the documentation site.

**Pull request merged**